### PR TITLE
Refactor Column to Status to go with back-end

### PR DESCRIPTION
**Pull request by Cireimu**
# Description
A supplementary branch to reflect back-end changes, which refactored everything that had to do with column to status.  
Back-End https://github.com/Lambda-School-Labs/mission-control-be/pull/105
Fixes # (issue)

## Type of change

Please delete options that are not relevant.

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Change Status

- [x] Complete, tested, ready to review and merge
- [ ] Complete, but not tested (may need new tests)
- [ ] Incomplete/work-in-progress, PR is for discussion/feedback

# Checklist

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] My code has been reviewed by at least one peer
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] There are no merge conflicts

### CODE REVIEW

**Review by D-Fink — APPROVED** [review]

**Review by abdirahmanfarah — APPROVED** [review]

File: src/components/Settings/ColumnSettings/index.jsx [hunk 1]
```diff
@@ -23,7 +23,7 @@ const ColumnSettings = () => {
     <div>
       <div className={columnEditCont}>
         {data
-          ? data.programs[0].columns.map(column => (
+          ? data.programs[0].statuses.map(column => (
```
Comment (Trip4077) [inline src/components/Settings/ColumnSettings/index.jsx:21]: This conditional may need to be tweaked as it's only checking that data exists, but not if programs contains any information, for example

```
data = { programs: null }
data = {  programs: undefined }
data = { something_else: value }
```

will all pass true on `if( data )` but will return `Cannot read map of undefined` or `__ has no property map`

Comment (Trip4077) [reply src/components/Settings/ColumnSettings/index.jsx:21]: Also noticing that you're directly referencing index 0 so

`data = {  programs: [] }` will also throw an error will trying to access statuses

**Review by Trip4077 — APPROVED** [review]
Very nice work! Thank you for the time of going through to rename all of those variables and properties.

**Comment by Memitaru** [comment]
[Refactor Column Names to Status](https://trello.com/c/XMz8PTLj/1-refactor-column-names-to-status)

**Pull request merged**